src/main.js

```diff
@@ -256,15 +256,19 @@ export async function buildWebApp(targetName) {
 }
 
 export async function add(resourceType = 'target') {
+  let result = false
   if (resourceType === 'target') {
     await addTarget()
       .then(() => {
         console.log(chalk.greenBright('Target successfully added!'))
+        result = true
       })
       .catch((err) => {
         displayResult(err, 'An error has occurred when adding the target.')
+        result = false
```

Comment:
in this case, it makes sense to assign `err` to `result` variable, so if the test fail, it will fail with err message